<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWebEngineView" name="webView" native="true"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget{
	border-image: url(:/background.png);
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>661</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="styleSheet">
          <string notr="true">#widget_2{
	border-image: url(:/CA.png);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar::item{
	border: 0px;
}</string>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionOpenView">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionCloseView">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionPauseView">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionUnpauseView">
   <property name="text">
    <string>继续</string>
   </property>
  </action>
  <action name="actionWeatherSetting">
   <property name="text">
    <string>设定天气</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>?</string>
   </property>
  </action>
  <action name="actionCameraControl">
   <property name="text">
    <string>监视相机控制</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
